spec/requests/order_items_spec.rb

```diff
@@ -4,6 +4,12 @@
   let!(:order_item) { create(:order_item, :order_id => order.id, :portfolio_item_id => portfolio_item.id, :tenant_id => tenant.id) }
   let(:portfolio_item) { create(:portfolio_item, :service_offering_ref => "123", :tenant_id => tenant.id) }
 
+  around do |example|
```

Comment:
@lindgrenj6 I don't think you need this, we are trying to get rid of the around blocks. Once you set the default_headers in the request and have each record have the :tenant_id => tenant.id it will do the trick.